test/test_mocks.py

```diff
@@ -207,7 +210,7 @@ def test_graph_factory_with_override_globals(gdcmodels, gdcdictionary):
 
     nodes = [
         dict(label='foo', node_id='id_1'),
-        dict(label='foo', node_id='id_2', baz='allowed_2', bar='hello', fobble=30),
+        dict(label='foo', node_id='id_2', baz='allowed_2', bar='hello', fobble=30, ages=[1,2]),
```

Comment:
We will mostly be using [string arrays](https://github.com/NCI-GDC/gdcdictionary/blob/72a123891347845dc1a8609558aa688176869078/gdcdictionary/schemas/diagnosis.yaml#L18071-L18091), as that is the only array supported by the dictionary at this time. So maybe add another test that covers the [string properties](https://github.com/NCI-GDC/psqlgraph/blob/01921c96d59ba08a16644f1173b115e318c50a0f/test/models.py#L35). You may have extend the dummy dictionary to use `enum` instead of free form string